syntax = "proto3";

package users;

// User Request
message UserRequest {
    int32 user_id = 1;
}

// User Response
message UserResponse {
    int32 user_id = 1;
    string username = 2;
    string email = 3;
    string first_name = 4;
    string last_name = 5;
}

// User Service
service UserService {
    rpc GetUser(UserRequest) returns (UserResponse);
}
